### PR TITLE
remove non compiling code

### PR DESCRIPTION
**Pull request by greole**
This PR just removes some parts of the code that wont work with ginkgo 1.8.0 it is not meant to produce a working  of OGL but rather allows to implement on top of it.

### CODE REVIEW

**Comment by chihtaw** [comment]
I found some checks failed. Is it okay to fail to build OGL for OpenFOAM?

**Comment by greole** [comment]
> I found some checks failed. Is it okay to fail to build OGL for OpenFOAM?

yes that is expected for now. We should fix that in a separate PR in this stack.

File: .github/workflows/static_checks.yaml [hunk 1]
```diff
@@ -43,40 +48,33 @@ jobs:
...
+           jq
     - uses: actions/download-artifact@v4
       with:
         name: build-artifact
         path: build
-
     - name: Run clang-tidy
       run: |
+        git config --global --add safe.directory /__w/OGL/OGL
         # Create list of .cpp files belonging to this repository
```
Comment (chihtaw) [inline .github/workflows/static_checks.yaml]: The comment may need to reflect the change:
`# Create list of .C files belonging to this repository`

File: .github/workflows/static_checks.yaml [hunk 2]
```diff
@@ -43,40 +48,33 @@ jobs:
...
-        git ls-files | grep -E "\.(cpp)" > pattern
+        git ls-files | grep -E "\.(C)" > pattern
         # Create list of .cpp files that are in this repository and part of the
```
Comment (chihtaw) [inline .github/workflows/static_checks.yaml:70]: `# Create list of .C files that are in this repository and part of the`

File: .github/workflows/static_checks.yaml [hunk 3]
```diff
@@ -43,40 +48,33 @@ jobs:
...
         # compilation database
         # also filters out " at the begin and end of the filename
-        jq ".[] | .file" build/ReleaseAll/compile_commands.json \
+        jq ".[] | .file" build/ReleaseCPUOnly/compile_commands.json \
```
Comment (chihtaw) [inline .github/workflows/static_checks.yaml:73]: Does `ReleaseAll` mean it is for both CPU and GPU?
Why changes to `ReleaseCPUOnly`?

Comment (greole) [reply .github/workflows/static_checks.yaml:73]: No, the `ReleaseAll` came from the corresponding NeoFOAM workflow and just means build all targets (unit tests, core, ...)

**Pull request merged**